browser/resources/settings/brave_leo_assistant_page/model_list_section.html

```diff
@@ -85,22 +110,31 @@
       >
         <div class="model">
           <div class="model-info">
-            <div class="label">[[item.displayName]]</div>
+            <div class="model-name-container">
+              <div class="label">[[item.displayName]]</div>
+            </div>
```

Comment:
what is up with the style change of model name?

Reply:
simplified, initial version was a bit more complicated https://github.com/brave/brave-core/pull/31570/commits/0d2e728b4dfdbbfec858edc6fbf25842e08a250c